### PR TITLE
feat: added investors page

### PR DESCRIPTION
**Pull request by JonasBne**

### CODE REVIEW

File: src/pages/investors.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <main>
+      <Helmet
+        titleTemplate="%s - Credit investing democratized"
+        link={[
+          { rel: 'shortcut icon', type: 'image/png', href: `${fav}` }
+        ]}
+        meta={[
+          {
+            name: 'description',
+            content: 'CREDIX is a decentralized credit marketplace connecting investors with FinTechs in emerging markets. Learn more about the future of debt financing now.'
```
Comment (Mathijs003) [inline src/pages/investors.tsx:37]: add a todo, we need to change the description

File: src/components/investors/HomeComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import { CredixButton } from '../shared/buttons/CredixButton';
+import SvgIcon, { SVG_ICONS } from '../shared/svg-icon/SvgIcon';
+import { IWidthProps } from '../../pages';
+import { StaticImage } from 'gatsby-plugin-image';
+
+// todo: fix url for becoming investor button
+export const HomeComponent = ({ isMobile }: IWidthProps) => {
```
Comment (Mathijs003) [inline src/components/investors/HomeComponent.tsx:8]: There is already a HomeComponent, rename to InvestorsHomeComponent or so
```suggestion
export const InvestorsHomeComponent = ({ isMobile }: IWidthProps) => {
```

File: src/components/investors/CTAComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { CTADivider } from '../shared/dividers/CTADivider';
+import { openInNewTab } from '../../pages';
+
+export const CTAComponent = () => {
```
Comment (Mathijs003) [inline src/components/investors/CTAComponent.tsx:5]: same for this, I would use a unique name for the InvestorsCTAComponent
```suggestion
export const InvestorsCTAComponent = () => {
```

File: src/theme/scss/components/_home-component-investors.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+  @extend .global-padding-left-right;
+
+  @include mobile {
+    min-height: 400px;
+    height: 500px;
+    max-height: 80vh;
+  }
+
+  &--arrow-row {
```
Comment (Mathijs003) [inline src/theme/scss/components/_home-component-investors.scss:14]: where in the home investors component is this arrow used?

File: src/theme/scss/components/_home-component-investors.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    justify-content: center;
+    margin-top: auto;
+    margin-bottom: 80px;
+    z-index: $Z_INDEX_ARROW;
+  }
+
+  &--arrow {
```
Comment (Mathijs003) [inline src/theme/scss/components/_home-component-investors.scss:21]: same

**Review by Mathijs003 — CHANGES_REQUESTED** [review]
fix comments

File: src/components/HeaderComponent.tsx [hunk 6]
```diff
@@ -8,10 +8,13 @@ export const HeaderComponent = ({ isMobile }: IWidthProps) => {
   const [isMenuOpen, setMenuOpen] = useState<boolean>();
 
   const headerLinks = () => {
+      // todo: fix urls
```
Comment (Mathijs003) [inline src/components/HeaderComponent.tsx:11]: https://www.gatsbyjs.com/docs/linking-between-pages/

like this : 

`<Link className="header-component--link"  to="/investors/">Investors</Link>`

But then probably we also need to overwrite the styling again

Comment (JonasBne) [reply src/components/HeaderComponent.tsx:11]: done on borrowers branch

**Review by Mathijs003 — APPROVED** [review]

**Pull request merged**